packages/calcite-components/stencil.config.ts

```diff
@@ -80,6 +81,11 @@ export const create: () => Config = () => ({
     { components: ["calcite-value-list", "calcite-value-list-item"] }
   ],
   outputTargets: [
+    reactOutputTarget({
```

Comment:
Can you create a follow-up issue to explore ways to build output targets separately?

Reply:
#7111